crates/nu-command/src/math/mod.rs

```diff
@@ -4,6 +4,7 @@ mod arccosh;
 mod arcsin;
 mod arcsinh;
 mod arctan;
+mod arctan2;
```

Comment:
You will also have to reexport the command struct like seen below (e.g. something like `MathArcTan2`) to make it useable.

Also the command needs to be explictly added to the `create_default_context()` function so it is part of the builtins shipped with nushell.

See for example https://github.com/nushell/nushell/pull/7409/files